scipy/interpolate/_rgi.py

```diff
@@ -217,20 +217,41 @@ def __init__(self, points, values, method="linear", bounds_error=True,
             self._validate_grid_dimensions(points, method)
         self.method = method
         self.bounds_error = bounds_error
+        # todo: sort checks in order of complexity and runtime, raising early
```

Comment:
Minor:This is something you are planning to do in this PR, or is it done and the comment should be removed already?

Reply:
Done